target/ppc/translate/storage-ctrl-impl.c.inc

```diff
@@ -35,6 +35,20 @@ static bool trans_SLBIE(DisasContext *ctx, arg_SLBIE *a)
     return true;
 }
 
+static bool trans_SLBIEG(DisasContext *ctx, arg_SLBIEG *a)
+{
+    REQUIRE_64BIT(ctx);
+    REQUIRE_INSNS_FLAGS2(ctx, ISA300);
+    REQUIRE_SV(ctx);
+
+#if !defined(CONFIG_USER_ONLY) && defined(TARGET_PPC64)
+    gen_helper_slbieg(cpu_env, cpu_gpr[a->rb]);
```

Comment:
ditto

Reply:
Fixed in 83f0f3e7bfad3dfa22a822e9a01ffbb56ed26518